packages/framework/ai-collab/src/aiCollabUiDiffApi.ts

```diff
@@ -0,0 +1,194 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+// The following interfaces are intended to be used to visualize the changes made to a SharedTree (as a result of using ai-collab) on a UI.
+
+/**
+ * A base interface to enforce consisntencty between all UiDiff objects.
+ * @remarks This object is not intended to be used directly.
+ * The union type UiDiff provides a better typescript experience
+ *
+ * @alpha
+ */
+export interface UiDiffBase {
```

Comment:
Nit: these diffs aren't really exclusive to "UI". I wonder if we should be a bit less prescriptive in the naming here. I wonder if we should just omit "Ui" from the type names.

Reply:
Make sense. Remove it. I think for now, this type only used by frond-end page but it could be used somewhere else in the future